packages/falso/src/lib/between-date.ts

```diff
@@ -13,21 +13,30 @@ interface BetweenOptions extends FakeOptions {
  *
  * @example
  *
- * randBetweenDate({ from: Date, to: Date })
+ * randBetweenDate({ from: new Date('10/07/1988'), to: new Date('10/07/1989') })
  *
  * @example
  *
- * randBetweenDate({ from: Date, to: Date, length: 10 })
+ * randBetweenDate({ from: new Date('10/07/1988'), to: new Date('10/07/1989'), length: 10 })
  *
  */
 export function randBetweenDate<Options extends BetweenOptions = never>(
   options: Options
 ) {
-  const from = options.from.getTime();
-  const to = options.to.getTime();
+  // TODO: RS - For now I have added default dates to fix the docs. We should look into ways to use the examples from the JSDocs for LIVE EDITOR functions
+  const from = options?.from?.getTime() ?? new Date('10/07/1988').getTime();
+  const to = options?.to?.getTime() ?? new Date('11/07/1989').getTime();
+
+  if (!from) {
+    throw new Error('From date must be passed');
+  }
+
+  if (!to) {
+    throw new Error('To date must be passed');
+  }
```

Comment:
These are redundant since it's enforced by the options type, the reason it happened in the docs it's because of the way we generate it.

Reply:
I have removed the default values and kept the thrown errors as I think default values in a between are a bit useless. Users could just use one of the other random date if they don't want to specify a `from` and `to`.

This won't error in the docs anymore now that i've added it to the `functionModifiers` object.

Reply:
But we have typescript protection, why do we need to throw?

Reply:
Fair point 👍